### DISCUSSION TITLE
Datadog Integration

### DISCUSSION THREAD

**Discussion by twdgn22** [gobuffalo/buffalo#2373] [Q&A]
Hi guys!
Is it any integrations library or something like that with APM - Datadog?

**Comment by sio4** [comment]
Hi @twdgn22 

Easy support for APMs could be nice but unfortunately, there is no Buffalo-specific module for supporting them yet. Actually, supporting that kind of integration is one of my key agenda for the next generation, but the plan is also not yet solid.

Suggestions and Ideas are always welcome!
